src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp() {
+        String[] Machine;
+        String Truck = new String();
+        String Bulldozer = new String();
+        String Excavator = new String();
```

Comment:
why do you create strings here? Create machine array instead: new Machine[]{new Truck(), etc}